js/Common.js

```diff
@@ -1337,13 +1337,14 @@ CRM.strings = CRM.strings || {};
    * @return string
    */
   var currencyTemplate;
-  CRM.formatMoney = function(value, format) {
+  CRM.formatMoney = function(value, format, onlyNumber) {
```

Comment:
I think this new param ought to be 2nd and not 1st.
I seriously doubt any current code is calling this fn with 2 params so it should be fine.
Also don't forget to update the docblock when adding params.